crates/subspace-service/src/dsn.rs

```diff
@@ -159,6 +160,10 @@ where
         provider_storage,
         max_established_incoming_connections: dsn_config.max_in_connections,
         max_established_outgoing_connections: dsn_config.max_out_connections,
+        max_pending_incoming_connections: dsn_config.max_in_connections
+            * SWARM_PENDING_TO_ESTABLISHED_CONNECTIONS_FACTOR,
+        max_pending_outgoing_connections: dsn_config.max_out_connections
+            * SWARM_PENDING_TO_ESTABLISHED_CONNECTIONS_FACTOR,
```

Comment:
Should likely be separate parameters or factor should be exposed as its own CLI argument.